README.md

```diff
@@ -2,10 +2,10 @@
 
 This is the source for https://docs.snowplow.io/docs.
 
-* [Contributing](#contributing)
-* [How to preview locally](#how-to-preview-locally)
-* [Organizing content](#organizing-content)
-* [Formatting content](#formatting-content)
+- [Contributing](#contributing)
+- [How to preview locally](#how-to-preview-locally)
+- [Organizing content](#organizing-content)
+- [Formatting content](#formatting-content)
 
 ## Contributing
```

Comment:
Should there be something in this section about how you run and check formatting?